### ISSUE TITLE
[XML] Preferences page for XML for spaces and tabs

### ISSUE THREAD

**Issue by LorenzoBettini** [eclipse/wildwebdeveloper#872]
This is a follow-up of https://github.com/eclipse/wildwebdeveloper/issues/139 to ask for a preference page to specify whether to use spaces or tabs for the XML editor (possibly by allowing to specify also how many spaces to insert for indentation).

As requested by @angelozerr in https://github.com/eclipse/wildwebdeveloper/issues/139#issuecomment-1250143363 I opened this new issue.

Thanks in advance

**Comment by angelozerr** [comment]
Thanks @LorenzoBettini for reporting this issue. To support this issue it requires some changes on Eclipse Platform Text. The first step is to give the capability to custom spaces/tabs with .editorconfig.

After that I will work on this issue but it will requires some contribution to Eclipse Platform Text.

**Comment by LorenzoBettini** [comment]
@angelozerr, thank you for your effort. In the meantime, is there a way to have two spaces in the XML file while still having tabs in the Java editor? I didn't find any satisfactory solution at the moment, but maybe I'm missing something...

**Comment by angelozerr** [comment]
Java Editor can customize that, no?

![image](https://user-images.githubusercontent.com/1932211/191977664-cfab8aeb-3662-4cec-9da6-5a5e64f9714d.png)

**Comment by LorenzoBettini** [comment]
But that requires defining a new custom profile, which, especially across workspaces, is a few lengthy operation..

**Comment by angelozerr** [comment]
> But that requires defining a new custom profile, which, especially across workspaces, is a few lengthy operation..

I agree but for the moment it is the only solution. Please wait for .editorconfig support at first.

**Comment by LorenzoBettini** [comment]
Actually, for the moment, I'm using a "replace all" of a tab with two spaces... ;)

**Comment by angelozerr** [comment]
Wow we need really provide .editorconfig support.

**Comment by persicsb** [comment]
Almost 3 years passed, and we still cannot configure the XML formatter for whitespace.

**Comment by mickaelistria** [comment]
@persicsb Please submit a good pull request that provides it and it will be reviewed within a few hours.
